joint_limits_interface/include/joint_limits_interface/joint_limits.h

```diff
@@ -50,6 +50,26 @@ struct JointLimits
       angle_wraparound(false)
   {}
 
+public:
+
+  // Helper function for debuggging, not realtime safe
+  void print()
+  {
+    std::cout << "min_position " << min_position << std::endl;
```

Comment:
pedantic: I'd only use `std::endl` in the last line, and use `"\n"` instead for all the others, to avoid flushing for each line: http://en.cppreference.com/w/cpp/io/manip/endl